src/exec/cmd.c

```diff
@@ -0,0 +1,68 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   cmd.c                                              :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: mhoyer <mhoyer@student.42.fr>              +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/07/22 09:52:56 by mhoyer            #+#    #+#             */
+/*   Updated: 2023/07/24 16:10:53 by mhoyer           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "exec.h"
+
+void	dup_in(t_command *cmd)
+{
+	int	fdin;
+
+	if (cmd->has_heredoc == 0)
```

Comment:
```suggestion
	if (cmd->has_heredoc == true)
```
remplace tout tes 1 0 par true false (include stdbool.h)